configuration/ldap/ldap_config.py

```diff
@@ -59,21 +59,28 @@ def _import_group_type(group_type_name):
 #     ldap.set_option(ldap.OPT_X_TLS_CACERTFILE, LDAP_CA_CERT_FILE)
 LDAP_CA_CERT_FILE = environ.get('LDAP_CA_CERT_FILE', None)
 
-AUTH_LDAP_USER_SEARCH_BASEDN = environ.get('AUTH_LDAP_USER_SEARCH_BASEDN', '')
-AUTH_LDAP_USER_SEARCH_ATTR = environ.get('AUTH_LDAP_USER_SEARCH_ATTR', 'sAMAccountName')
+AUTH_LDAP_USER_SEARCH_BASEDN = environ.get("AUTH_LDAP_USER_SEARCH_BASEDN", "")
+AUTH_LDAP_USER_SEARCH_ATTR = environ.get("AUTH_LDAP_USER_SEARCH_ATTR", "sAMAccountName")
+AUTH_LDAP_USER_SEARCH_FILTER: str = environ.get(
+    "AUTH_LDAP_USER_SEARCH_FILTER", f"({AUTH_LDAP_USER_SEARCH_ATTR}=%(user)s"
```

Comment:
The closing ) is missing in the default `AUTH_LDAP_USER_SEARCH_FILTER` at the end.

Reply:
done in 084e9c6